app/src/main/res/values/strings.xml

```diff
@@ -28,7 +28,7 @@
     <string name="create_key">Create new key</string>
     <string name="show_key">Show key</string>
     <string name="verify_sig">Verify signature</string>
-    <string name="storage_permission_explanation">Give permission for storage!</string>
+    <string name="storage_permission_explanation">Please give storage permissions for scanning the text on the passport</string>
```

Comment:
I believe this explanation is re-used in some places

Reply:
Ok, I created a separate one for the ocr.